slack_bolt/version.py

```diff
@@ -1,2 +1,2 @@
 """Check the latest version at https://pypi.org/project/slack-bolt/"""
-__version__ = "1.9.4"
+__version__ = "1.10.0a"
```

Comment:
We'll bump the minor version as this pull request introduces new functionalities